### DISCUSSION TITLE
Access

### DISCUSSION THREAD

**Discussion by ontargetWeb** [trobfab/fabrik#538] [General]
@trob, @skurvish @joomlahenk  Would you consider giving me access to the Private repo, I'm doing my best to help you all with getting F4 stable. I have 2 pretty complex projects which are using a large number of elements, list plugins etc so I can give you a lot of feedback. I don't want to waste your time reporting stuff that might have already been fixed. Having access to the repo will allow me to overwrite my files more completely. E.g i just added a new commit which actually made something stop working so I don't know if there were other commits i missed which were dependent. I wouldn't have the skills to start my own fork of fabrik so my role is purely an active contributor and bug reporter to your much appreciated work.
Sincerely
onTarget

**Comment by joomlahenk** [comment]
@ontargetWeb :If you are using many plugins, sure you can really help us to make F4 even better. Please report any issues or change proposal you have in the issue section here. 
I don't understand why you would need access to the private repo just to overwrite your files more completely.
You can download F4 as explaned in the wiki, then you have all files available just the same as we have in the repo. You can do your tests on any of these files, whatever you want to do. There are no other "secret" files or tools the we use on the repo.

**Reply by ontargetWeb** [reply]
Ok thanks for the reply
